source/styleEditor.js

```diff
@@ -1,3 +1,17 @@
 $(document).ready(function(){
-  // your code goes here.
+ var selector = $('input[name="selector"]');
```

Comment:
A helpful convention, when setting variables to jQuery elements, is to prepend the variable with `$`. E.g., `$selector`.